src/components/chip-set/chip-set.tsx

```diff
@@ -248,6 +258,15 @@ export class ChipSet {
         this.handleDeleteAllIconClick =
             this.handleDeleteAllIconClick.bind(this);
         this.renderDelimiter = this.renderDelimiter.bind(this);
+        if (!this.autocomplete) {
+            this.autocomplete = this.getAutocompleteValue();
```

Comment:
Props are immutable so we can not assign a new value like this. We can however assign it when its declared